src/js/utils/can-autoplay.js

```diff
@@ -33,12 +33,12 @@ import Promise from '../polyfills/promise';
 
 function startPlayback (element, { muted }) {
     // Configure element.
     element.muted = muted;
-    element.src = VIDEO;
```

Comment:
Oh nice. Let's see if we can use `URL` now. I think we couldn't in v7 because of IE9, but we should be good now. There are some player utils which could benefit from this.